src/TributeRange.js

```diff
@@ -589,18 +589,16 @@ class TributeRange {
             coordinates.right = windowWidth - rect.left - windowLeft
         }
 
-        let parentHeight = this.tribute.menuContainer
-            ? this.tribute.menuContainer.offsetHeight
-            : this.getDocument().body.offsetHeight
-
+        let menuContainer = this.tribute.menuContainer
         if (menuIsOffScreen.bottom) {
-            let parentRect = this.tribute.menuContainer
-                ? this.tribute.menuContainer.getBoundingClientRect()
-                : this.getDocument().body.getBoundingClientRect()
-            let scrollStillAvailable = parentHeight - (windowHeight - parentRect.top)
-
             coordinates.top = 'auto'
-            coordinates.bottom = scrollStillAvailable + (windowHeight - rect.top)
+            if (menuContainer) {
```

Comment:
Note that I did not touch the semantics in 1) the case when `menuContainer` **is** defined (related refactoring/clean-up only).

The changes here changes the semantics **only** for 2) the case when `menuContainer` is **not** defined.

There are also some issues with the 1) case, but I did not want to touch it here since I'm unsure about the proper fix.

But fix for the second case works nicely. Please check the mentioned Codepens.